src/main/java/cc/hyperium/config/Category.java

```diff
@@ -2,21 +2,21 @@
 
 public enum Category {
 
-    //Settings Tab
+    // Settings Tab
     GENERAL("General"),
     IMPROVEMENTS("Improvements"),
     INTEGRATIONS("Integrations"),
     COSMETICS("Cosmetics"),
+    CUSTOMIZATION("Customization"),
```

Comment:
Whats the point of this tab?

Reply:
removed it in a different pr, but its so i could get all the customization stuff out of misc bc it was getting packed

Reply:
oh ok